partcad/src/partcad/shape.py

```
@@ -417,13 +417,26 @@ async def render_step_async(
             step_opts, filepath = self.render_getopts("step", ".step", project, filepath)
 
-            cq_obj = await self.get_cadquery()
+            obj = await self.get_wrapped()
 
             def do_render_step():
-                nonlocal project, filepath, cq_obj
+                nonlocal project, filepath, obj
+                from OCP.STEPControl import STEPControl_Writer, STEPControl_AsIs
+                from OCP.Interface import Interface_Static
+
                 if not project is None:
```

Ruff (0.8.2)

Test for object identity should be `is not`

Convert to `is not`

(E714)